src/redux/actionTypes.js

```diff
@@ -0,0 +1,8 @@
+// CONTAINS PREDEFINED ACTION TYPES FOR CLARITY.
+
+// Books
+export const ADD_BOOK = 'ADD_BOOK';
+export const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// Categories
+export const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] The value of each action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:       const ADD_BOOK = bookstore/books/ADD_BOOK and const REMOVE_BOOK = bookstore/books/REMOVE_BOOK

Reply:
@EricMbouwe  Thanks for the review, and I assure you that I will apply these changes as soon as possible.